tools/flang2/flang2exe/semutil0.cpp

```diff
@@ -742,6 +848,16 @@ cngcon(INT oldval, DTYPE oldtyp, DTYPE newtyp)
     } else if (from == TY_DBLE) {
       num[0] = oldval;
       num[1] = stb.dbl0;
+    } else if (from == TY_QUAD) {
```

Comment:
The macro `TARGET_SUPPORTS_QUADFP` is missing here.

Reply:
Fixed